### PR TITLE
Fix next steps style

### PR DESCRIPTION
**Pull request by AGuyNamedMarco**
styling fixes for the Next Steps section in the Home and About pages.

### CODE REVIEW

File: src/_scss/pages/about/nextSteps.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
     @import "mixins/fullSectionWrap";
     @include fullSectionWrap(0, 0);
     & .block-wrap {
-        @include span-columns(16);
+        @include span-columns(4);
```
Comment (mab6bf) [inline src/_scss/pages/about/nextSteps.scss]: This breaks the mobile view

File: src/_scss/pages/homepage/_linksSectionBox.scss [hunk 2]
```diff
@@ -1,7 +1,10 @@
+@import "../../mixins/elementContainerWrap";
+
 @mixin linksSectionBox {
     .links-section-box {
-        @include span-columns(16);
-        border: 1px solid $color-gray;
+        @include display(flex);
+        @include span-columns(4);
```
Comment (mab6bf) [inline src/_scss/pages/homepage/_linksSectionBox.scss]: This breaks the mobile view

File: src/js/components/about/NextStepsBox.jsx [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
 import React from 'react';
 import * as AboutIcons from 'components/sharedComponents/icons/about/AboutIcons';
+import * as Icons from 'components/sharedComponents/icons/Icons';
```
Comment (mab6bf) [inline src/js/components/about/NextStepsBox.jsx]: Please remove this if we aren't using it anywhere. This is an unused import, which is failing the linter.

File: src/js/components/homepage/LinksSectionBox.jsx [hunk 4]
```diff
@@ -44,27 +44,31 @@ export default class LinksSectionBox extends React.Component {
...
         }
 
         return (
             <div className="links-section-box">
                 {this.switchIcon()}
-                <h4 className={`links-section-box-primary-text${disabledStatus}`}>
-                    {this.props.text}
-                </h4>
+                <h6 className={`links-section-box-primary-text${disabledStatus}`}>
+                    {this.props.title}
```
Comment (mab6bf) [inline src/js/components/homepage/LinksSectionBox.jsx]: Please add `title: React.PropTypes.string` to the `propTypes`

File: src/js/components/homepage/LinksSectionBox.jsx [hunk 5]
```diff
@@ -44,27 +44,31 @@ export default class LinksSectionBox extends React.Component {
...
+                </h6>
+                <p className={`links-section-box-secondary-text${disabledStatus}`}>
+                    {this.props.subtitle}
```
Comment (mab6bf) [inline src/js/components/homepage/LinksSectionBox.jsx]: Please add `subtitle: React.PropTypes.string` to the `propTypes`

**Comment by AGuyNamedMarco** [comment]
All issues corrected!  Thanks for the comments, Mike.

**Pull request merged**